CONTRIBUTING.md

```diff
@@ -0,0 +1,56 @@
+# Contributing Guidelines
+
+## Notes
+
+- **Before spending lots of time on something, ask for feedback on your idea first!**
+- Please search issues and pull requests before adding something new to avoid duplicating efforts and conversations.
+- Issues which are considered to be easy to solve are marked with the label `easy`: [Browse all easy issues](https://github.com/LibrePCB/LibrePCB/labels/easy).
+
+## Getting Started
+
+- Make sure you have a [GitHub account](https://github.com/signup/free).
+- Open a new issue for your idea, assuming one does not already exist.
+- Fork the repository on GitHub.
+- Have a look at our [development resources](https://github.com/LibrePCB/LibrePCB/tree/master/dev), 
+  especially at the [Doxygen documentation](http://librepcb.org/LibrePCB-Doxygen/master/).
+- When using QtCreator, import and use our [code style guide file](https://github.com/LibrePCB/LibrePCB/blob/master/dev/CodingStyle_QtCreator.xml).
+
+## Making Changes
+
+- Create a topic branch from where you want to base your work.
+  - This is usually the master branch.
+  - To quickly create a topic branch based on master: 
+    `git checkout -b my_contribution master`
+  - Please avoid working directly on the `master` branch.
+- Write code which follows our [code style guides](http://librepcb.org/LibrePCB-Doxygen/master/df/d24/doc_code_style_guide.html)
+  and [.editorconfig settings](https://github.com/LibrePCB/LibrePCB/blob/master/.editorconfig).
+- Make commits of logical units.
+  - Make sure your commit messages are in the proper format.
+    Example: [0d896b69fdffbeb873957ed2b66eb9b0aa2ba750](https://github.com/LibrePCB/LibrePCB/commit/0d896b69fdffbeb873957ed2b66eb9b0aa2ba750)
+- Make sure you have added the necessary tests for your changes.
+- Run _all_ the tests to assure nothing else was accidentally broken.
```

Comment:
I agree, but it should be worded as an useful hint, not as a "command"/recommendation